src/js/playlist/playlist.js

```diff
@@ -45,11 +44,17 @@ export function normalizePlaylistItem(model, item, feedData) {
     
     playlistItem.feedData = feedData;
 
-    return playlistItem;
+    return formatItem(playlistItem);
 }
 
 export const fixSources = (item, model) => filterSources(formatSources(item, model), model.getProviders());
 
+function formatItem(item) {
+    const liveSyncDuration = item.sources[0].liveSyncDuration;
+    item.dvrSeekLimit = item.liveSyncDuration = liveSyncDuration;
```

Comment:
I'm concerned that this will overwrite any existing `dvrSeekLimit` set by users.

Unfortunately it doesn't look like we have any tests in commercial that do this (all the test configs set it to 25).

Reply:
This definitely will. It will always at least overwrite it to the default 25, so that's a good thing to note.